src/utils/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export const getArticles = () => {
+  return axios
+    .get("https://portfolio-web-service-n7kk.onrender.com/api/articles")
```

Comment:
It would be nice to use axios.create to create an axios instance, this will make things easier as you go through the project